user-guide/Reference/DataMiner_Tools/BrokerGateway_Migration.md

```diff
@@ -135,9 +145,11 @@ The following actions will be executed during the migration, in the indicated or
 
 ## Migrating back to the old system
 
-After you have migrated to BrokerGateway, to go back to the old system, follow the same procedure as when you [run the migration](#migrating-to-brokergateway), but use the `uninstall` command instead of the `install` command.
+After you have migrated to BrokerGateway, there are two different ways you can go back to the old system:
+
+- Download and run the package [NATSMigrationRollback.dmupgrade](https://community.dataminer.services/download/natsmigrationrollback-dmupgrade/).
```

Comment:
I've added the info about the rollback package here. Could you check if this is OK?

Reply:
The manual method is actually mentioned on line 152, as an alternative to this line 😊